test/jdk/java/foreign/StdLibTest.java

```diff
@@ -383,7 +383,7 @@ enum PrintfArg {
         STRING(MemorySegment.class, C_POINTER, "%s", arena -> {
             return arena.allocateUtf8String("str");
         }, "str"),
-        CHAR(byte.class, C_CHAR, "%c", arena -> (byte) 'h', 'h'),
+        CHAR(int.class, C_INT, "%c", arena -> (int) 'h', (int) 'h'), // promote to int, per C spec
```

Comment:
is it important to retain this, given that there's already an INT case?

Reply:
Yes, good idea.